README.md

```diff
@@ -1,13 +1,55 @@
-There are a lot of different ways to organize a workshop, but if you've got the [Minimum Viable Workshop](/Minimum-Viable-Workshop.md), you'll have a great time. There are way more people in the world who are excited to teach and learn Clojure than you know.
+# Get involved
 
-The [Workshop Planning Timeline](Workshop-Planning-Timeline.md) is a good place to start; it's divided into sections before, during, and after the workshop. There are *a lot* of individual tasks, so we recommend organizing in a team.
+ClojureBridge aims to increase diversity within the Clojure community by offering free, beginner-friendly Clojure programming workshops for women.
 
-We have past organizers who can help you figure out what to do. Email info@clojurebridge.org and we'll introduce you to an organizing mentor!
+ClojureBridge was started in 2014 and is part of [Bridge Foundry](http://bridgefoundry.org/). All around the world there have been workshops organized by people like you! Many people volunteered in different ways. Here are some insights for you to get involved:
 
-[Resources on Diversity](Resources-on-Diversity.md) provides a number of great resources on how to create a diverse, inclusive event.
+## What can you do?
 
-ClojureBridge follows the BridgeFoundry [Code of
-Conduct](http://bridgefoundry.org/code-of-conduct/).
+There are different ways you can contribute. Maybe you like to...
+
+###...organize!
+
+You want to organize a workshop in your area? Great! Let’s get started! For a [minimal viable workshop](https://github.com/ClojureBridge/organizing/blob/master/Minimum-Viable-Workshop.md) you need to find: 
+
+- **A venue:** A safe and quiet space for learners to do their first steps in Clojure. An office leisure space or a co-working space might work.
+- **A date:** ClojureBridge is a 2 day workshop. Which means you need to find a date for the install party (e.g. Friday evening from 5PM - 8PM) and a date for the workshop day (e.g. Saturday 9 AM - 6 PM).

```

Comment:
I suggest _two-day_ in place of _2 day_.

Let's make the spacing around "PM" consistent. :stuck_out_tongue_winking_eye: